### PR TITLE
LUN-2833: Add logistics methodology page

### PR DESCRIPTION
**Pull request by rbruggem**
For dashboard logged-out users, the page contains some visible content,
some blurred content (but still present and in clear text)  and some
encrypted content.

Upon login/signup the blurred and encrypted content becomes visible.

The encrypted content is decrypted using the `docs_publishable_key`
cookie set by the dashboard.

This change includes two components. For dashboard logged-out users:
* `LoginWall`, blurs its children and displays a login button, that
  redirects to the dashboad for login
* `LoginSink`, hides its children

Screenshot 2023-10-10 at 18 12 52

### CODE REVIEW

File: docs/product-and-design-guides/logistics-calculations-methodology.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+<div className="paragraphSections">
+
+<div>
+
+## 1. Shipment method
+
+Lune covers all modes of transport including ocean, inland waterways, road, air, and rail. In addition, you can also calculate warehouse-related emissions.
+
+Depending on the data you have available, the shipment method used for the calculation can be either high-level or more granular. For high-level data (e.g. if you only know the shipment was using a truck or a container ship), we use industry average [Emission Factors](/product-and-design-guides/emission-factors) by the GLEC Framework.
```
Comment (charcarr) [inline docs/product-and-design-guides/logistics-calculations-methodology.md:90]: we use industry average -> we use an industry average by the GLEC Framework

File: src/components/LoginWall.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+        <Box
+            sx={{
+                position: 'relative',
+            }}
+        >
+            {!loggedIn && (
+                <Box
+                    sx={{
+                        backdropFilter: 'blur(4px)',
```
Comment (charcarr) [inline src/components/LoginWall.tsx:28]: ou cool I was wondering how this happened.

**Review by charcarr — APPROVED** [review]
Cool, excited to see this.

File: docs/product-and-design-guides/logistics-calculations-methodology.md (hunk 1 above)
Comment (jstasiak) [reply docs/product-and-design-guides/logistics-calculations-methodology.md:90]: Even when "industry average" is followed by "Emission Factors" here?

**Pull request merged**